### ISSUE TITLE
[Doc] [Serve] Custom Metrics Tutotial

### ISSUE THREAD

**Issue by simon-mo** [ray-project/ray#12545]
@maxpumperla  mentioned it might be nice to have a serve doc pages about custom metrics.

I think we can start with the following topics:
- which metrics is exported by default
- how to export custom metric (get the backend name to tag along the metrics)
- when to use metrics and when to use structured logging

cc @edoakes

**Comment by maxpumperla** [comment]
@simon-mo yes, that'd be great, thank you! The current tutorial here https://docs.ray.io/en/latest/ray-metrics.html works with `ray start` to expose a metrics port. I'd really like to see what the intended usage is specifically for a ray serve example.
